src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocBlockTagLocationCheck.java

```diff
@@ -0,0 +1,177 @@
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import com.puppycrawl.tools.checkstyle.StatelessCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailNode;
+import com.puppycrawl.tools.checkstyle.api.JavadocTokenTypes;
+
+/**
+ * <p>
+ * Checks that a
+ * <a href="https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#block-tags">
+ * javadoc block tag</a> appears only at the beginning of a line, ignoring leading asterisks and
+ * white space.
+ * </p>
+ * <p>
+ * Rationale: all javadoc block tags not placed at the beginning of a line are
+ * treated as the plain text. This may be intentional, but changing the text may
+ * unexpectedly place them at the beginning of the line and they will no longer
+ * be recognized as plain text. Conversely, if a block tag is accidentally moved
+ * to the middle of a line, it will be treated as plain text without any
+ * violations, but the generated documentation will be erroneous.
+ * </p>
+ * <p>
+ * To place a tag explicitly as text, escape the {@code @} symbol with &amp;#64;
+ * or place it inside {@code {@code }}, for example:
+ * </p>
+ * <pre>
+ * &#47;**
+ *  * &amp;#64;serial literal in {&#64;code &#64;serial} Javadoc tag.
+ *  *&#47;
+ * </pre>
+ * <ul>
+ * <li>
+ * Property {@code tags} - Specify the javadoc tags to process.
+ * Default value is {@code exception, hidden, author, deprecated, throws, version, see,
+ * serialData, param, serial, provides, serialField, uses, return, since}.
+ * </li>
+ * <li>
+ * Property {@code violateExecutionOnNonTightHtml} - Control when to print violations
+ * if the Javadoc being examined by this check violates the tight html rules defined at
+ * <a href="https://checkstyle.org/writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.
+ * Default value is {@code false}.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the default check:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocBlockTagLocation"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * &#47;**
+ *  * Escaped tag &amp;#64;version (OK)
+ *  * Plain text with {&#64;code &#64;see} (OK)
+ *  * A @custom tag (OK)
+ *  * &#64;since 1.0 (OK)
+ *  * text &#64;return (violation)
+ *  * * &#64;param (violation)
+ * +* &#64;serial (violation)
+ *  * &#64;see first (OK) &#64;see second (violation)
+ *  *&#47;
+ * public int field;
+ * </pre>
+ * <p>
+ * To configure the check to verify tokens from
+ * <a href="https://openjdk.java.net/jeps/8068562">JEP 8068562</a>:
```

Comment:
in addition to default or instead ?
lets make two examples of how to configure Check to extend defaults and how to override defaults.

Reply:
done